edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test/LoggerTest.cs

```diff
@@ -46,6 +46,33 @@ LoggerConfiguration TestSinkMap(LoggerSinkConfiguration loggerSinkConfiguration)
             Assert.Equal("3", emittedEvents[0].Properties["Severity"].ToString());
         }
 
+        [Fact]
+        public void NoNewLinesTest()
+        {
+            using (var writer = new System.IO.StringWriter())
```

Comment:
nit: import System.IO
```suggestion
            using (var writer = new tringWriter())
```